tests/common/util.rs

```diff
@@ -1333,6 +1333,59 @@ pub fn expected_result(ts: &TestScenario, args: &[&str]) -> std::result::Result<
     ))
 }
 
+/// This is a convenience wrapper to run a ucmd with root permissions.
```

Comment:
```suggestion
/// This is a convenience wrapper to run a ucmd with root permissions.
/// It can be used to test programs when being root is needed
```